Sources/TestingMacros/Support/Additions/MacroExpansionContextAdditions.swift

```diff
@@ -80,30 +80,60 @@ extension MacroExpansionContext {
 // MARK: -
 
 extension MacroExpansionContext {
+  /// Whether or not our generated warnings are suppressed in the current
+  /// lexical context.
+  ///
+  /// The value of this property is `true` if the current lexical context
+  /// contains a node with the `@_semantics("testing.macros.nowarnings")`
+  /// attribute applied to it.
+  ///
+  /// - Warning: This functionality is not part of the public interface of the
+  ///   testing library. It may be modified or removed in a future update.
+  var areWarningsSuppressed: Bool {
+    for lexicalContext in self.lexicalContext {
```

Comment:
Disable this in release builds so folks don't try to use it!

Reply:
Clever